plugins/modules/dedicated_server_install.py

```diff
@@ -94,11 +100,17 @@ def run_module():
     if template not in compatible_templates["ovh"] and template not in compatible_templates["personal"]:
         module.fail_json(msg="f{template} doesn't exist in compatibles templates")
 
+    if raid == 'enabled':
+        no_raid = False
+    elif raid == 'disabled':
+        no_raid = True
+
     details = {"details":
                {"language": "en",
                 "customHostname": hostname,
                 "sshKeyName": ssh_key_name,
-                "softRaidDevices": soft_raid_devices}
+                "softRaidDevices": soft_raid_devices,
+                "noRaid": no_raid }
```

Comment:
The linter complains on this line: `E202 whitespace before '}'`
```suggestion
                "noRaid": no_raid}
```